docs/migration.md

```diff
@@ -2,96 +2,253 @@
 
-Your feedback will be a critical part of ensuring that we have made the right tradeoffs with the API changes in V2.
+We believe Pydantic V2 is ready for your production applications. This page provides a guide highlighting the most
+important changes to help you migrate your code from Pydantic V1 to Pydantic V2.
 
-To get started with the Pydantic V2 alpha, install it from PyPI.
+## Install Pydantic V2 beta
+
+Your feedback will be a critical part of ensuring that we have made the right tradeoffs with the API changes in
+Pydantic V2.
+
+To get started with the Pydantic V2 beta, install it from PyPI.
 We recommend using a virtual environment to isolate your testing environment:
 
 ```bash
-pip install --pre -U "pydantic>=2.0a1"
+pip install --pre -U "pydantic>=2.0b1"
 ```
 
-!!! warning "Alpha release!"
-    Note that there are still some rough edges and incomplete features, and while trying out the Pydantic V2 alpha releases you may experience errors.
-
-    We encourage you to try out the alpha releases in a test environment and not in production.
-    Some features are still in development, and we will continue to make changes to the API.
+!!! warning "Beta release!"
+    Note that there are still some rough edges, and while trying out the Pydantic V2 beta releases you may experience errors.
+
+    We encourage you to try out the beta releases in a test environment and not in production.
+    While we do not currently intend to make changes to the API, it's still possible that we
+    will discover problems requiring breaking changes.
+
+If you do encounter any issues, please [create an issue in GitHub](https://github.com/pydantic/pydantic/issues) using
+the `bug V2` label. This will help us to actively monitor and track errors, and to continue to improve the library's
+performance.
+
+## Migration guide
+
+### Changes to `pydantic.BaseModel`
+
+Various method names have been changed; all non-deprecated `BaseModel` methods now have names matching either the
+format `model_.*` or `__.*pydantic.*__`. Where possible, we have retained the deprecated methods with their old names
+to help ease migration, but calling them will emit `DeprecationWarning`s.
+
+| Pydantic V2  | Pydantic V1 |
+| ------------ | ----------- |
+| `model_dump()` | `dict()` |
+| `model_dump_json()` | `json()` |
+| `model_copy()` | `copy()` |
+| `model_json_schema()` | `json_schema()` |
+| `model_construct()` | `construct()` |
+| `model_validate()` | `parse_obj()` |
+| `__pydantic_validator__` | `__validators__` |
+| `model_fields` | `__fields__` |
+| `__signature__` | `__signature__` |
+| `__class_vars__` | `__class_vars__` |
+| `__pydantic_private__` | `__private_attributes__` |
+
+* Some of the built-in data-loading functionality has been slated for removal. In particular,
+    `parse_raw` and `parse_file` are now deprecated. In Prefect V2, `model_validate_json` works like `parse_raw`. Otherwise, you should load the data and then pass it to `model_validate`.
```

Comment:
Almost! 👀 
```suggestion
    `parse_raw` and `parse_file` are now deprecated. In Pydantic V2, `model_validate_json` works like `parse_raw`. Otherwise, you should load the data and then pass it to `model_validate`.
```